src/c++/library/CMakeLists.txt

```diff
@@ -477,10 +475,8 @@ if(TRITON_ENABLE_CC_HTTP)
     endif() # TRITON_ENABLE_GPU
 
     if(TRITON_ENABLE_GPU)
-      target_link_libraries(
-        ${_client_target}
-        PUBLIC CUDA::cudart
-      )
+      target_include_directories(${_client_target}
+                                 PUBLIC ${CUDAToolkit_INCLUDE_DIRS})
```

Comment:
This is the same change that was reverted in https://github.com/triton-inference-server/client/pull/799